broker/applications/osb-broker/test/acceptance/service-broker-api.GetEndpoints.spec.js

```diff
@@ -0,0 +1,597 @@
+'use strict';
+
+const _ = require('lodash');
+const parseUrl = require('url').parse;
+const app = require('../../../../test/test_broker/support/apps').internal;
+const { catalog } = require('@sf/models');
+const config = require('@sf/app-config');
+const {
+  CONST,
+  commonFunctions: {
+    encodeBase64
+  }
+} = require('@sf/common-utils');
+const camelcaseKeys = require('camelcase-keys');
+
+describe('service-broker-api-enhancement', function () {
```

Comment:
Can you pls follow the hierarchy in [service-broker-api-2.0.instances.director.spec.js](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/9aaa417e70f5237bdcb80f65b8c0dd42e67a75e3/broker/applications/osb-broker/test/acceptance/service-broker-api-2.0.instances.director.spec.js) . For example, hierarchical position for tests for provision operation is: `describe('service-broker-api-2.0') --> describe('director') --> describe('#provision')`

Reply:
@jayeshjk Done.